relay-server/src/metrics_extraction/spans/mod.rs

```diff
@@ -17,9 +16,9 @@ use crate::statsd::RelayTimers;
 /// This configuration is temporarily hard-coded here. It will later be moved to project config
 /// and be provided by the upstream.
 static SPAN_EXTRACTION_CONFIG: Lazy<MetricExtractionConfig> = Lazy::new(|| {
-    MetricExtractionConfig::new(
```

Comment:
I'm not sure if I prefer the constructor here.

Reply:
I removed it for two reasons:
 1. This is a data struct that already had at least one public field and has to have a public signature. In that case, we can construct directly.
 2. The usage (right in the line below) had comments to name the fields. That indicates that the constructor has a bad signature.